### ISSUE TITLE
Tabbed UI breaks position 'sidebar'

### ISSUE THREAD

**Issue by TimHal** [payloadcms/plugin-seo#16]
Using the SEO plugin and enabling it for a collection breaks positioning elements in the admin sidebar for that collection.

**Reproduce**
- Install plugin
- Use a minimal config with `tabbedUI` enabled
- Observe that the target collection can not show any fields in the sidebar.

```
 seo({
      collections: ['users'],
      tabbedUI: true,
    }),
```
Disabling the plugin for that collection or setting `tabbedUI` to false fixes the problem immediately.

**Comment by TimHal** [comment]
I want to add: This might rather be a problem with the payload admin frontend than with this plugin, but I am yet to confirm that, leaving the issue here for now if thats ok.

**Comment by ChristopherNowlan** [comment]
I can confirm I have the same issue. Setting TabbedUI to false also resolves the issue for me.

**Comment by jacobsfletch** [comment]
Hey @TimHal and @ChristopherNowlan I can confirm that this is core behavior of Payload. A field with `position: 'sidebar'` that is within a tab _does not appear in the sidebar_ but instead, renders in place. When enabling `tabbedUI`, the plugin takes all of the fields in your collection and renders them within a `tabs` field.

We basically have two options here, which is to either design an API that will only render _some_ of your fields in the tab and keeps the other top-level (I think this is messy), or we support sidebar fields from a tab in Payload core (preferred).

On a similar note, I fixed bug related to tabs in [this PR](https://github.com/payloadcms/plugin-seo/pull/25).

Closing for now at least.
